packages/fxa-graphql-api/src/gql/account.resolver.ts

```diff
@@ -518,9 +500,12 @@ export class AccountResolver {
     @Args('input', { type: () => AccountResetInput })
     input: AccountResetInput
   ): Promise<AccountResetPayload> {
+    // Switch the API instance we used based on the payload.
+    // const authAPI = input.newPasswordV2 ? this.authApiV2 : this.authAPI;
```

Comment:
Was this commented out for testing?

Reply:
Yes, I was just doing some sanity checks during testing. The 'V2' instance just had key stretch version set to 2. It's actually backwards compatiable, so we can just use this config everywhere.